Resources/Prototypes/_Den/Actions/vampire.yml

```
@@ -0,0 +1,45 @@
+      sprite: _Den/Interface/Actions/actions_vampire.rsi
+      state: glare
+    useDelay: 1.0
+    itemIconStyle: BigAction
+    sound: !type:SoundPathSpecifier
+      path: /Audio/_Den/Effects/blind.ogg
+  - type: AutoRecharge
+    rechargeDuration: 20
+  - type: LimitedCharges
+    maxCharges: 2
+  - type: InstantAction
+    event: !type:VampireGlareAbility
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)